Syllabus.md

```diff
@@ -192,7 +205,7 @@ We will explore LSST data visualization in a bit more detail. This session will
   + [Source measurement tutorial](https://pipelines.lsst.io/getting-started/photometry.html)
   + [Kron.ipynb](https://github.com/RobertLuptonTheGood/notebooks/blob/2eeee8b9fe35077387485e488c965f1ea3d39418/Demos/Kron.ipynb)
 
-<a name="astrometry"></a>(9) Astrometric calibration
+<a name="astrometry"></a>10. Astrometric calibration
```

Comment:
Both the Source/Object measurement and Astrometric calibration are labeled section "10".